typescript-loops-practice/add-suffix-to-all.ts

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words: string[], suffix: string): any {
+  const suffArr: string[] = [];
+  for (let i = 0; i < words.length; i++) {
+    suffArr[i] = words[i] + suffix;
```

Comment:
suggested to call .push() method when adding new elements to an array, and use bracket syntax array[...] = ... when modifying existing elements...